pandas/tests/groupby/test_timegrouper.py

```diff
@@ -717,7 +717,7 @@ def test_groupby_groups_periods(self):
 
     def test_groupby_first_datetime64(self):
         df = DataFrame([(1, 1351036800000000000), (2, 1351036800000000000)])
-        df[1] = df[1].view("M8[ns]")
+        df[1] = df[1]._values.view("M8[ns]")
```

Comment:
```suggestion
        df[1] = df[1].astype("M8[ns]")
```

(I don't think there is a reason for the view here?)